server/consumer.go

```diff
@@ -55,6 +55,8 @@ type ConsumerInfo struct {
 	NumPending     uint64          `json:"num_pending"`
 	Cluster        *ClusterInfo    `json:"cluster,omitempty"`
 	PushBound      bool            `json:"push_bound,omitempty"`
+	// TimeStamp indicates when the info was gathered
+	TimeStamp time.Time `json:"time_stamp"`
```

Comment:
We have `"first_ts"` and `"last_ts"` being passed around so would prefer shorter version too here:
```suggestion
	TimeStamp time.Time `json:"ts"`
```